src/invokers.ts

```diff
@@ -82,7 +81,7 @@ export function makeResolverInvoker<T>(resolver: Resolver<T>) {
    * @param  {MethodName} methodToInvoke
```

Comment:
Just to not be inconsistent:

```suggestion
   * @param  methodToInvoke
```